recipe/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - setuptools
   run:
     - python
+    - packaging
+    - pytoolconfig
```

Comment:
Needs to be at least `1.2.2`: https://github.com/python-rope/rope/blob/1.7.0/pyproject.toml#L24